Basic-Car-Maintenance/Shared/Utilities/Constants.swift

```diff
@@ -16,3 +16,34 @@ enum FirestoreCollection {
 enum FirestoreField {
     static let userID = "userID"
 }
+
+enum SFSymbol {
+    //Tabview icons
+    static let dashboard = "list.dash.header.rectangle"
+    static let odometer = "gauge"
+    static let settings = "gear"
```

Comment:
```suggestion
    static let gear = "gear"
```

I want to keep this gear, because I know which it is, and it would be more confusing to me, to call this settings